openapi.yaml

```diff
@@ -1,279 +1,678 @@
+        adaParkingSpaceCount:
+          description: 'Count of ADA parking spaces if the location is parking'
+          type: integer
+        motorcycleParkingSpaceCount:
+          description: 'Count of motorcycle parking spaces if the location is parking.'
+          type: integer
+        weeklyMenu:
+          description: 'Link to a weekly menu if the location is dining.'
+          type: string
+          format: url
+          example: https://app.uhds.oregonstate.edu/api/dining/weeklymenu/index?loc=1
+    ServicesAttributes:
+      properties:
+        name:
+          type: string
+          description: Name of the Service.
+        tags:
+          description: 'Tags associated with this location'
+          type: array
+          items:
             type: string
-            format: uri
-            description: A link to further information about the error
-            example: https://developer.oregonstate.edu/documentation/error-reference#1234
-  ErrorResult:
-    properties:
-      errors:
-        type: array
-        items:
-          $ref: '#/definitions/ErrorObject'
+            example: recreation
+        openHours:
+          type: array
+          items:
+            $ref: '#/components/schemas/DayHours'
+    RelationshipsInLocations:
+      properties:
+        services:
+            $ref: '#/components/schemas/ServicesRelationship'
+    RelationshipsInServices:
+      properties:
+        locations:
+            $ref: '#/components/schemas/ServicesRelationship'
+    ServicesRelationship:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ResourceObjectIdentifier'
+    ResourceObjectIdentifier:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+    DayHours:
+      description: >
+        This object represents open hours for a location, relative to the date
+        that the API returns the response. 1 represents the current day,
+        2 represents the next day, etc. All times are in UTC.
+      properties:
+        1:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        2:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        3:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        4:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        5:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        6:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+        7:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+    TimeSlot:
+      properties:
+        start:
+          type: string
+          format: date-time
+        end:
+          type: string
+          format: date-time
+    Links:
```

Comment:
Try naming variables/fields/functions more descriptive. Also, you should directly use the definition from here: https://github.com/osu-mist/express-api-skeleton/blob/openapi3/openapi.yaml#L198